doc/htmldoc/index.rst

```diff
@@ -1,16 +1,139 @@
-NEST Simulator documentaiton
+NEST Simulator documentation
 ============================
 
+
+Welcome!
+--------
+
+
+.. grid::
+
+  .. grid-item::
+
+    NEST is used in computational neuroscience to model and study behavior of large networks of neurons.
+
+    The models describe single neuron and synapse behavior and their connections.
+    Different mechanisms of plasticity can be used to investigate artificial learning
```

Comment:
```suggestion
    Different models of plasticity can be used to investigate the mechanisms learning
```
possibly, this is a "standard text" paragraph and edits should go in somewhere upstream instead?